### ISSUE TITLE
HiddenInput fields are visible

### ISSUE THREAD

**Issue by alexgleason** [wagtail/wagtail#4943]
### Issue Summary

Wagtail's admin CSS doesn't properly style hidden inputs. See the screenshot (the second field, "Entity key").

![screenshot from 2018-12-05 11 24 01](https://user-images.githubusercontent.com/3639540/49527805-a7d42280-f880-11e8-8de8-8b18f2e7f468.png)

In markup, it's `<input type="hidden">` which is fine, but the label and the whole div still show up.

### Steps to Reproduce

```python
class MyForm(forms.Form):
    tip_content = forms.CharField()
    entity_key = forms.CharField(required=False, widget=forms.HiddenInput)
```

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: **no**

### Further info

I've worked around this with the following CSS:

```css
.field.hidden_input {
  display: none;
}
```

**Comment by gasman** [comment]
Hi @alexgleason,
Is this form rendered with your own template? If so, you should loop over `visible_fields` and `hidden_fields` separately ([see Django docs](https://docs.djangoproject.com/en/2.1/topics/forms/#looping-over-hidden-and-visible-fields)), and only use the Wagtail styled includes for the visible ones. As far as I'm aware Wagtail's own forms always do this (see for example https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templates/wagtailadmin/generic/create.html) - if that's not the case, then that's a bug we should fix.

**Comment by alexgleason** [comment]
Thanks for the info. I'm experiencing some separate issues now but I'll close this.

After switching to this:

```html
<div class="nice-padding">
  <form class="draftailmodal" action="/admin/draftailmodal/chooser/" method="post" novalidate>
    {% csrf_token %}
    {% for field in form.hidden_fields %}{{ field }}{% endfor %}
    <ul class="fields">
      {% for field in form.visible_fields %}
        {% include "wagtailadmin/shared/field_as_li.html" %}
      {% endfor %}
      <li><input type="submit" value="{% trans 'Save' %}" class="button"></li>
    </ul>
  </form>
</div>
```
Draftail breaks out of the ul tag  and the labels disappear.

![screenshot from 2018-12-05 11 49 14](https://user-images.githubusercontent.com/3639540/49529425-f931e100-f883-11e8-8f36-f94fb63752bd.png)

Sticking with my hacked CSS for now. I may open a separate issue for the Draftail thing if I can figure out what's actually happening.

Thank you!

**Comment by gasman** [comment]
Ah, it's entirely possible that the `visible_fields` / `hidden_fields` mechanism is treating the Draftail widget as a hidden field, because that's what the underlying `<input>` field is. Our chooser widget classes define `is_hidden = False` to override this, and the Draftail widget may well need this added too...

https://github.com/wagtail/wagtail/blob/a9f33aa99eae37824eb604b6146f7faaa523bbfa/wagtail/admin/widgets.py#L116-L118

**Comment by gasman** [comment]
Fixed in #5583
